clang/test/CodeGen/bit-int-ubsan.c

```diff
@@ -3,9 +3,7 @@
 
 // The runtime test checking the _BitInt ubsan feature is located in compiler-rt/test/ubsan/TestCases/Integer/bit-int.c
 
-#include <stdint.h>
-#include <stdio.h>
-
+typedef unsigned int uint32_t;
```

Comment:
Can we fix the RUN line while we're at it? It's using `%clang` which runs the Clang driver, it should be using `%clang_cc1` so it only executes the frontend.

Reply:
Makes sense. Will update.